python/elastic_search_serialization.py

```diff
@@ -0,0 +1,622 @@
+import json
+from typing import Any, List, Dict, Tuple
+from datetime import datetime
+
+import elasticsearch
+from elasticsearch.helpers import streaming_bulk, parallel_bulk
+
+import mgp
+
+
+# Elasticsearch constants
+ACTION = "action"
+INDEX = "index"
+ID = "_id"
+SOURCE = "source"
+_SOURCE = "_source"
+SETTINGS = "settings"
+NUMBER_OF_SHARDS = "number_of_shards"
+NUMBER_OF_REPLICAS = "number_of_replicas"
+MAPPINGS = "mappings"
+DYNAMIC_TEMPLATES = "dynamic_templates"
+MAPPING = "mapping"
+ANALYZER = "analyzer"
+STRING = "string"
+EVENT_TYPE = "event_type"
+CREATED_VERTEX = "created_vertex"
+CREATED_EDGE = "created_edge"
+VERTEX = "vertex"
+EDGE = "edge"
+INDEX_TYPE = "index_type"
+AGGREGATIONS = "aggregations"
+HITS = "hits"
+TOTAL = "total"
+
+# Linkurious constants
```

Comment:
I think we should remove anything that has connection with clients

Reply:
Yes, this was left accidently, I removed other things.